### PR TITLE
i2c: Add Stringer for i2c.Addr

### PR DESCRIPTION
**Pull request by NeuralSpaz**
Add Addr.String()
Add Flag example

i2c.Addr was missing String() to complete flag.Value interface.

### CODE REVIEW

File: conn/i2c/i2c.go [hunk 1]
```diff
@@ -125,6 +125,11 @@ func (a *Addr) Set(s string) error {
 	return nil
 }
 
+// String returns s i2c.Addr as a string formated in hexadecimal.
```
Comment (maruel) [inline conn/i2c/i2c.go]: returns

File: conn/i2c/i2c_test.go [hunk 2]
```diff
@@ -128,3 +128,21 @@ func TestAddr_Set(t *testing.T) {
...
+		Addr Addr
+		want string
+	}{
+		{0x01, "0x1"},
+		{0x24, "0x24"},
+		{0x3ff, "0x3ff"},
+	}
+
+	for _, tt := range tests {
+		got := tt.Addr.String()
```
Comment (maruel) [inline conn/i2c/i2c_test.go]: This can be inlined in the next line as `if got := tt.Addr.String(); got != tt.want {`

File: conn/i2c/i2c.go (hunk 1 above)
Comment (maruel) [inline conn/i2c/i2c.go]: a :)

Comment (NeuralSpaz) [reply conn/i2c/i2c.go]: an :)

**Pull request merged**